src/lib/workspace/Variable.tsx

```diff
@@ -37,7 +39,18 @@ export function VariableDetails(props: Props) {
           <div className={cx('-ProviderLabelPrefix')}>
             Original variable name:
           </div>
-          &nbsp;{variable.providerLabel}&nbsp;
+          &nbsp;
```

Comment:
It's a minor pain, but I'd pluralise "Original variable name" for the multifilter.